Add doc dir for Keptn 0.16 (#1228)
keptn/keptn.github.io: content/docs/0.16.x/automated_operations/action-provider/index.md

---
title: Action-Provider
description: Add an action-provider to execute custom remediation actions.
weight: 5
icon: setup
keywords: [0.16.x-automated-operations]
---

> Depending on the action that should be executed in course of a remediation (or operational), a corresponding action-provider must be deployed. This action-provider receives a Keptn event, performs its action, and notifies Keptn about the execution. 

Reviewer: ```suggestion
Depending on the action that should be executed in the course of a remediation (or operational), a corresponding action-provider must be deployed. This action-provider receives a Keptn event, performs its action, and notifies Keptn about the execution. 
```

> To plug-in an action-provider into a remediation, the remediation config must be extended by an action supported by the provider. Consequently, Keptn - as control plane and responsible for orchestrating the remediation - can send out the proper event for the action-provider.  

Reviewer: ```suggestion
To plug an action-provider into a remediation, the remediation configuration must be extended by an action that the provider supports. Consequently, Keptn - as control plane and responsible for orchestrating the remediation - can send out the proper event for the action-provider.  
```
